tests/cloudformation/graph/checks/test_yaml_policies.py

```diff
@@ -86,8 +86,10 @@ def assert_entities(self, expected_entities: List[str], results: List[Record], a
             self.assertTrue(found, f"expected to find entity {expected_entity}, {'passed' if assertion else 'failed'}")
 
     def assert_evaluated_keys(self, expected_evaluated_keys: List[str], results: List[Record]):
-        for record in results:
-            self.assertEqual(expected_evaluated_keys, record.check_result["evaluated_keys"])
+        expected_evaluated_keys.sort()
+        evaluated_keys_results = results[0].check_result["evaluated_keys"]
+        evaluated_keys_results.sort()
+        self.assertListEqual(expected_evaluated_keys, evaluated_keys_results)
```

Comment:
```suggestion
        evaluated_keys_results = results[0].check_result["evaluated_keys"]
        self.assertCountEqual(expected_evaluated_keys, evaluated_keys_results)
```
this checks if they are same, but ignores ordering 🙂